department-of-veterans-affairs/abd-vro
# Hot fix test

Merged Lighthouse API service change in Main branch, and fix related unit tests

## What was the problem?
We modified lighthouse fhir api call in main branch to improve the performance. 
1. The unit tests need to be adjusted.  
2. We need to merge the change into develop branch.

## How does this fix it?[^1]
Cherry pick the related change in main branch and fix the unit tests.

## How to test this PR
- Step 1. Build the app and launch it.
- Step 2. Test Health data assessment endpoint.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
